expat/CMakeLists.txt

```diff
@@ -611,6 +611,60 @@ else()
 endif()
 string(TOUPPER "${CMAKE_BUILD_TYPE}" _EXPAT_BUILD_TYPE_UPPER)
 
+#
+# packaging   
+#
+include(InstallRequiredSystemLibraries)
+
+set(CPACK_GENERATOR TGZ ZIP)
+if(WIN32 AND NOT CYGWIN)
+  set(CPACK_GENERATOR NSIS ${CPACK_GENERATOR})
+endif(WIN32 AND NOT CYGWIN)
+if(APPLE)
+  set(CPACK_GENERATOR DragNDrop ${CPACK_GENERATOR})
+endif(APPLE)
+if (UNIX AND NOT APPLE AND NOT WIN32)
+  set(CPACK_GENERATOR DEB ${CPACK_GENERATOR})
+endif (UNIX AND NOT APPLE AND NOT WIN32)
```

Comment:
Debian already has packaging for Expat, a more proper one from the point of Debian. So I'm not sure we want to have two "competing" Debian packages. Did you think of some specific gain why this would be important to have?

Reply:
No, lets remove this package generator, just here because used on source project.